examples/Remedy_Examples/inlineMemoryRemedy_samesa.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: activemonitor.keikoproj.io/v1alpha1
+kind: HealthCheck
+metadata:
+  generateName: fail-healthcheck-
+  namespace: health
+spec:
+  repeatAfterSec: 60 # duration in seconds
+  level: cluster
+  workflow:
+    generateName: fail-workflow-
+    resource:
+      namespace: health # workflow will be submitted in this ns
+      serviceAccount: activemonitor-healthcheck-sa # workflow will be submitted using this
+      source:
+        inline: |
+            apiVersion: argoproj.io/v1alpha1
+            kind: Workflow
+            spec:
+              ttlSecondsAfterFinished: 60
+              entrypoint: start
+              templates:
+              - name: start
+                retryStrategy:
+                  limit: 1
+                container: 
+                  image: ravihari/ctrmemory:v2
+                  command: ["python"]
+                  args: ["promanalysis.py", "http://prometheus.system.svc.cluster.local:9090", "health", "memory-demo", "memory-demo-ctr", "95"]
```

Comment:
Should we add some documentation about the required pre-requisites for this example to run?

E.g. It needs prometheus to be running in the same namespace as this workflow and should be using port 9090.

Reply:
sure.. will add the documentation for example